packages/util/src/BaseApi.ts

```diff
@@ -438,6 +450,12 @@ export class BaseApi {
         return this.api.tx.pswapDistribution.claimIncentive()
       case Operation.ClaimExternalRewards:
         return this.api.tx.rewards.claim('0xa8811ca9a2f65a4e21bd82a1e121f2a7f0f94006d0d4bcacf50016aef0b67765692bb7a06367365f13a521ec129c260451a682e658048729ff514e77e4cdffab1b') // signature mock
+      case Operation.ReferralReserveXor:
+        return this.api.tx.referrals.reserve('0')
+      case Operation.ReferralUnreserveXor:
+        return this.api.tx.referrals.unreserve('0')
+      case Operation.ReferralSetInvitedUser:
+        return this.api.tx.referrals.setReferrer('')
```

Comment:
Will this extrinsic work with empty account address? I mean we can insert here some mock of account address to be sure for extrinsic success

Reply:
Yes, it works without any mock address.